Game/space-invaders/src/Game.cpp

```diff
@@ -32,6 +33,8 @@ void Game::Init()
     m_PlayerManager->CreatePlayer(WIDTH, HEIGHT);
 
     m_Level = std::make_unique<GameLevel>(800, 600, *this);
```

Comment:
Isn't a change from this MR, but we could change the 800 and 600 for `WIDTH` and `HEIGHT`